projects/src/simbicycle.hh

```diff
@@ -112,6 +113,7 @@ void Bicycle<Model, Observer, Haptic>::update_dynamics(real_t roll_torque_input,
         m_observer.set_state(x);
     }
 
+    m_input = input;
```

Comment:
I guess I was worried accessing it from another thread but I'm not doing a good job about it anyways.